teek/_structures.py

```diff
@@ -475,15 +476,21 @@ def runner(*junk):
 
             command = teek.create_command(runner, [str, str, str])
             teek.tcl_call(None, 'trace', 'add', 'variable',
-                        self, 'write', command)
+                          self, 'write', command)
 
         return self._write_trace
 
 
-class StringVar(TclVariable): type_spec = str       # flake8: noqa
-class IntVar(TclVariable): type_spec = int          # flake8: noqa
-class FloatVar(TclVariable): type_spec = float      # flake8: noqa
-class BooleanVar(TclVariable): type_spec = bool     # flake8: noqa
+class StringVar(TclVariable): type_spec = str
+
+
+class IntVar(TclVariable): type_spec = int
+
+
+class FloatVar(TclVariable): type_spec = float
+
+
+class BooleanVar(TclVariable): type_spec = bool
```

Comment:
This looks awful IMO. Can you just keep the noqa comments? This is just a small nit-pick though, so if you really want the code to look like this, then I'll merge the pull request anyway.